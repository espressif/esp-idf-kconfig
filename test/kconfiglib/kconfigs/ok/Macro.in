mainmenu "test using simple macros"


MAX_NUMBER_OF_MOTORS = 8

config MOTOR_NUMBER
    int "Number of motors"
    range 1 10
    default $(MAX_NUMBER_OF_MOTORS) # should expand to 8
    help
        Number of motors connected to the starship

config MOTOR_NUMBER_AS_STRING
    string "Number of motors as string"
    default "$(MAX_NUMBER_OF_MOTORS)" # should expand to "8"
    help
        Number of motors connected to the starship as a string

rsource "kconfigs_for_sourcing/macro"
