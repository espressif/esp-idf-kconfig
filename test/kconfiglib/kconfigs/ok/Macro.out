#
# Automatically generated file. DO NOT EDIT.
# Espressif IoT Development Framework (ESP-IDF)  Project Configuration
#
# default:
CONFIG_MOTOR_NUMBER=8
# default:
CONFIG_MOTOR_NUMBER_AS_STRING="8"
# default:
CONFIG_RANGED_CONFIG=50
